doc/design.md

```diff
@@ -0,0 +1,136 @@
+# agentmon Unmasked
+
+Fundamentally, `agentmon` is a silly little server that proxies metric
+reporting to a third party, namely, Heroku's Application Metrics endpoint
```

Comment:
semicolon before the transitional phrase `namely`? Also, after the transition you should form a sentence by using a pronoun, ... to a third party; namely, Heroku's Application Metrics endpoint **which is** available over HTTPS.